tools/cldr/cldr-to-icu/README.txt

```diff
@@ -0,0 +1,71 @@
+Basic instructions for running the LdmlConverter via Maven
+----------------------------------------------------------
+
+Note that these instructions do not currently support configuration of the converter for things
+such as limiting the set of files produced. That is supported in code and could be easily added
+to the binary, or encapsulated via an Ant task, but currently it is not directly supported.
+See the IcuConverterConfig class for the API by which this can be supported.
+
+
+Notes on JDK versions:
+----------------------
+
+It is possible that some JDK versions will not, by default, support the set of features necessary
+in the SAX parser (XML parser). If you see the message:
+
+[ERROR] ------------------------------------------------------------------
+[ERROR] Unknown error reading /<path-to-xml-file>
+[ERROR]
+[ERROR] This can sometimes be caused by using a version of Java which does
+[ERROR] not support all the XML parsing features required by this tool.
+[ERROR] Try setting the JAVA_HOME environment variable to a different Java
+[ERROR] release, if possible.
+[ERROR] ------------------------------------------------------------------
+
+Then you should see if it's possible to use a different Java runtime environment by setting
+JAVA_HOME environment variable. In Unix, this is possible by prefixing the command with a
+local assignment, such as:
+
+JAVA_HOME='<path-to-alternate-jvm>' mvn exec:java ...
+
+To run the converter tool, you need to know the locations of several key directories before
+hand. These are:
+
+
+Important directories:
+----------------------
+
+<CLDR_DIR>   = The root directory of the CLDR release.
+<ICU_DIR>    = The root directory of the ICU release (probably a parent directory of where
+               this README is located).
+<DTD_CACHE>  = The temporary cache directory in which DTD files are downloaded (this is the
+               same directory as would be used when running tools from the CLDR project).
+<OUT_DIR>    = The output directory into which ICU data files should be written.
```

Comment:
OUT_DIR also should be defaulted to the appropriate location relative to ICU_DIR

Reply:
This one I disagree on. The default behaviour of overwriting some, but maybe not all, existing files in the release directory seems very dangerous.

Reply:
I'll have to see what the overall workflow looks like then.

Reply:
Well, it's not my call, since I'm not doing the releases but a default behaviour of "overwrite a non-specific set of files in an existing directory" seems weird to me.

Reply:
that's actually been the usual workflow: `cd ~/src/icu/icu4c/source/data && ant clean && ant all`. Delete all generated data and rebuild. Verify, and check in.